README.md

```diff
@@ -2,24 +2,13 @@
 
 [![Join the chat at https://gitter.im/yelouafi/redux-saga](https://badges.gitter.im/yelouafi/redux-saga.svg)](https://gitter.im/yelouafi/redux-saga?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) [![npm version](https://img.shields.io/npm/v/redux-saga.svg?style=flat-square)](https://www.npmjs.com/package/redux-saga)
 
-An alternative Side Effects middleware (aka Asynchronous Actions) for Redux applications. Instead of dispatching Thunks which get handled by the `redux-thunk` middleware, you create *Sagas* to gather all your Side Effects logic in a central place.
+`redux-saga` is a library that aims to make asynchronous flows (e.g. data fetching) in React.js applications easier and better.
```

Comment:
Do you think we can borrow something from this http://survivejs.com/blog/redux-saga-interview/#redux-middlewares. Mainly, define what 'Side Effects' are in the context of Redux (async and impure calls).